js/ui/drop_down_editor/ui.drop_down_list.js

```diff
@@ -487,12 +487,14 @@ const DropDownList = DropDownEditor.inherit({
         this._setListFocusedElementOptionChange();
     },
 
-    _renderPreventBlur: function($target) {
+    _renderPreventBlur: function($target, exclusionElements = []) {
```

Comment:
Isn't it better to implement 2 different functions: one for ddList and another for  TagBox?

Why I think it's a good idea:
- second parameter is passed from inheritor to parent component (and it's useful only for 1/4 inheritors)
- new specific case is actual only for the TagBox
- we create event for TagBox on ddList namespace now

Reply:
done